specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/subnetResources.json

```diff
@@ -279,6 +284,7 @@
       "properties": {
         "id": {
           "type": "string",
+          "pattern": "^/subscriptions/[^/]*/resourceGroups/[^/]*/providers/Microsoft.(ClassicNetwork|Network)/virtualNetworks/[^/]*/subnets/[^/]*$",
```

Comment:
This regex allows double slashes? e.g. /subscriptions//resourceGroups/...

Reply:
Actually, this is the same as the patterns other RPs have defined. Examples:
azure-rest-api-specs\specification\redis\resource-manager\Microsoft.Cache\stable\2017-02-01\redis.json

azure-rest-api-specs\specification\apimanagement\resource-manager\Microsoft.ApiManagement\preview\2019-12-01-preview\apimdeployment.json